core/backend/src/test/imodel/Views.test.ts

```diff
@@ -0,0 +1,27 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import { expect } from "chai";
+import * as sinon from "sinon";
+import { CloudSqlite } from "../../CloudSqlite";
+import { IModelDb } from "../../IModelDb";
+
+describe("IModelDb.Views", () => {
+  afterEach(() => {
+    sinon.restore();
+  });
+
+  describe("accessViewStore", () => {
+    it("forwards requestToken and accessLevel to CloudSqlite", async () => {
```

Comment:
this test doesn't really prove much. Specifically it doesn't show that the userToken is passed and used from RPC. In fact, i'm not sure how the integration tests in `ViewStore.test.ts` work now, since the userToken from the frontend should be wrong. Previously they used the userToken stored in the backend. How does that work now?

Reply:
update: I now see that those tests just hardwire the userToken (which is why this problem existed unnoticed.) Let me look at how to make them work better.